src/scene.ts

```diff
@@ -4282,6 +4299,18 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         }
 
         this._frameId++;
+        this._defaultFrameBufferCleared = false;
+        if (this._activeCamera?.outputRenderTarget && !this.activeCamera?.isRigCamera) {
+            this._activeCamera.outputRenderTarget._cleared = false;
+        }

```

Comment:
Testing with the [default BabylonNative playground app](https://github.com/BabylonJS/BabylonNative/blob/master/Apps/Playground/Scripts/experience.js), I was seeing broken clearing behavior due to the rig camera output render targets not getting their `_cleared` flags reset. Adding this block fixed the behavior:
```javascript
if (!!this._activeCamera?.rigCameras?.length) {
    for (let i = 0; i < this._activeCamera.rigCameras.length; ++i) {
        const rtt = this._activeCamera.rigCameras[i].outputRenderTarget;
        if (rtt) {
            rtt._cleared = false;
        }
    }
}
```

There might be a better solution, just throwing this one out there.

I don't have my VR headset in the office right now (testing on HoloLens), but this should be reproducible on the x64 win32 build of BabylonNative `mkdir Build; cd Build; cmake ..`, set [`vr = true`](https://github.com/BabylonJS/BabylonNative/blob/master/Apps/Playground/Scripts/experience.js#L11) in experience.js.

Reply:
Thanks a lot, I added the lines you suggested.